### ISSUE TITLE
eye-hal does not compile on macOS

### ISSUE THREAD

**Issue by lewisfm** [raymanfx/eye-rs#9]
`eye-hal`, mentioned in the readme, does not compile on macOS.
```rs
error[E0308]: mismatched types
  --> .cargo/registry/src/index.crates.io-6f17d22bba15001f/eye-hal-0.1.0/src/platform/mod.rs:49:21
   |
49 |     fn default() -> Self {
   |        -------      ^^^^ expected `Context<'_>`, found `()`
   |        |
   |        implicitly returns `()` as its body has no tail or `return` expression

For more information about this error, try `rustc --explain E0308`.
error: could not compile `eye-hal` (lib) due to previous error
```

**Comment by raymanfx** [comment]
Hi, I just (finally) got around to giving this project some love and published `eye-hal 0.2.0` on crates.io. Please check that release - I verified it on my MacBook Pro (M1).

Streaming using the built-in iSight camera should work, as should USB devices.

**Comment by raymanfx** [comment]
Closing as there was no feedback and it works for me on Sonoma 14.2.1.
